src/main/java/de/tum/www1/orion/connector/ide/ArtemisConnector.java

```diff
@@ -1,7 +0,0 @@
-package de.tum.www1.orion.connector.ide;
-
-import netscape.javascript.JSObject;
-
-public interface ArtemisConnector {
-    void attachTo(JSObject jsObject, String memberName);
-}
```

Comment:
I think the JavaScriptConnector Interface serves a very similar purpose. Also since the attachTo method is no longer needed there is nothing to add to this ArtemisConnector interface.